MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include README.md
+recursive-include cmsplugin_feed_ai *.*
```

Comment:
Ok. Removed from the manifest and removed the `long_description` line in setup.py.